contracts/src/StakeTable.sol

```diff
@@ -241,7 +241,7 @@ contract StakeTable is Initializable, InitializedAt, OwnableUpgradeable, UUPSUpg
             revert ZeroAddress();
         }
         token = ERC20(_tokenAddress);
-        lightClient = LightClient(_lightClientAddress);
+        lightClient = ILightClient(_lightClientAddress);
```

Comment:
@alxiong @alysiahuggins another practical argument why interfaces are better. It seems that this is handled much better by testing tools. Still need to figure out what this does to the bytecode and if this is upgrade safe, and if not, how we should handle this.

Reply:
> another practical argument why interfaces are better. It seems that this is handled much better by testing tools.

didn't understand. 
1. wdym by "handled better by testing tool" here? what are we trying to achieve?
2. what other benefits you have in mind? 
3. what's the take away here?

Reply:
awesome! nice work @sveitser